src/orng.cpp

```diff
@@ -139,6 +149,13 @@ ACTION orng::requestrand(uint64_t assoc_id,
         rec.signing_value = signing_value;
         rec.caller = caller;
     });
+
+    // record the signing value in the old way for backwards compatibility with v1 dependant contracts
+    action(
```

Comment:
what happens if we send the same signing_value in different publicKey scope? This action might be failed?

Reply:
that's true under 2 conditions: 1. the user uses poor/non-random signing values, and 2. we have not had enough time to clear out the old signing values via `cleansigvals`.
If users are using random signing values then the chance of collision is super small. Like it should never happen
And if we manage to have already done `cleansigvals` for all of the previous keys then even if the user uses non random signing values it won't matter because that function clears the old signing values from self scope as well as the hashed key scope
So I am pretty sure we should be fine for this concern